lib/brainstem/presenter_collection.rb

```diff
@@ -275,7 +278,12 @@ def run_search(scope, includes, sort_name, direction, options)
         search_options[:page] = calculate_page(options)
       end
 
-      search_options.reverse_merge!(extract_filters(options))
+      extracted_filters = extract_filters(options)
+      extracted_filters_for_search = extracted_filters.each.with_object({}) do |(key, val), hash|
+        hash[key] = val[:arg] unless val[:arg].nil?
```

Comment:
Yeah, [this spec](https://github.com/mavenlink/brainstem/blob/add_params_to_filters/spec/brainstem/presenter_collection_spec.rb#L619) tests the situation where `val[:arg]` is nil..